plugins/modules/remote_management/manageiq/manageiq_tags.py

```diff
@@ -71,7 +71,7 @@
       url: 'http://127.0.0.1:3000'
       username: 'admin'
       password: 'smartvm'
-      validate_certs: false
+  register: result
```

Comment:
The `register: result` only makes sense for `_info` modules, or when the return value is of interest. I would not add it to random other module examples.

Reply:
(un)done